packages/design-system/src/components/FieldContainer/FieldContainer.tsx

```diff
@@ -97,23 +113,38 @@ export class FieldContainer extends React.Component<FieldContainerProps> {
     } = this.props;
 
     const ComponentType = component;
-    const classes =
-      ComponentType === 'fieldset' ? classNames('ds-c-fieldset', className) : className;
+    const isFieldset = ComponentType === 'fieldset'
+    const classes = classNames({
+      'ds-c-fieldset': isFieldset
+    }, className);
+    const bottomError = errorPlacement === 'bottom' && errorMessage;
 
     // Field input props handled by <FieldContainer>
-    const fieldInputProps = {
+    const fieldInputProps = { 
       id: this.id,
       labelId: this.labelId,
+      errorId: this.errorId,
       setRef: this.setFieldRef,
     };
 
+    const renderBottomError = () => {
```

Comment:
Since this is not used in a conditional block below, does it need to be a function? Could it just be a value?

Reply:
yep good catch!